src/read_csv.py

```diff
@@ -3,13 +3,16 @@
 import os
 
 
-def readFile(file):
-    countylist = []
-    newspaperlist = []
+def read_in_counties_newspapers(file):
+    """ A function that takes in a file of counties and the

```

Comment:
Replace docstring with `Takes in a csv file with each row containing a county and newspaper and outputs a list of (county, newspaper) tuples.`

Reply:
done